source/ProgArgs.cpp

```diff
@@ -2987,6 +3003,10 @@ void ProgArgs::printHelpS3()
             "S3 access secret. (This can also be set via the " S3_ENV_SECRET_KEY " env variable.)")
         (ARG_S3SESSION_TOKEN_LONG, bpo::value(&this->s3SessionToken),
              "S3 session token. (Optional. This can also be set via the " S3_ENV_SESSION_TOKEN " env variable.)")
+        (ARG_S3CREDFILE_LONG, bpo::value(&this->s3CredentialsFile),
```

Comment:
alphabetic ordering by "--optionname" so that the options appear in alphabetic order for the user running `elbencho --help-s3`